catalog/views.py

```diff
@@ -1,7 +1,20 @@
 from django.shortcuts import render
+from catalog.models import Product
 
 def home(request):
-    return render(request, 'index.html')
+    products = Product.objects.all()
+    context = {'products': products}
+    return render(request, 'index.html', context)
 
 def contacts(request):
     return render(request, 'contacts.html')
+
+def product_detail(request, product_id):
+    product = Product.objects.get(id=product_id)
```

Comment:
Если продукта по id в базе не найдешь, то сайт упадет с 500 ошибкой - это плохо. Правильнее отдать код 404 (NotFound), поддержка которого уже предусмотрена в django
```suggestion
    product = get_object_or_404(Product, id=product_id)
```